### ISSUE TITLE
Feature Suggestion: Demos

### ISSUE THREAD

**Issue by woojoo666** [qiao/PathFinding.js#110]
Great visualizer! Just a suggestion: it would be nice if there was a set of pre-made maps to choose from to quickly see how each pathfinder works in different scenarios

**Comment by rustinlee** [comment]
:+1: 

Sounds like this would need to be in two roughly equal parts, a new UI for selecting maps and a number (~5?) of different saved configurations.

**Comment by woojoo666** [comment]
yeah that sounds perfect. Not too sure how many presets there should be either, but perhaps just a couple that showcase the best and worst case for each algorithm

**Comment by rustinlee** [comment]
There has apparently already been a little bit of discussion on something similar: https://github.com/qiao/PathFinding.js/issues/9
